frontend_lib/src/index.js

```diff
@@ -91,6 +102,14 @@ export {
   NewMemberForm,
   CUSTOM_EVENT,
   BREADCRUMBS_TYPE,
+  FETCH_CONFIG,
```

Comment:
@gbonaspetti I would like you to put every debug data into a specific property `debug`.
This is to not pollute to namespace of frontend_lib with debug variables